index.d.ts

```diff
@@ -236,6 +236,11 @@ export declare interface TracerOptions {
    */
   experimental?: boolean | {
     b3?: boolean
+    /**
+     * Whether to write traces to log output, rather than send to an agent
+     * @default false
+     */
+    exporter?: 'log-exporter' | 'agent-exporter'
```

Comment:
Is it possible to use the values directly from the constants?

Reply:
I tried to import the constants form ext/exporters/.d.ts, and it wasn't playing nicely with the type check.